### PR TITLE
Replace travis CI with GitHub Actions

### PR DESCRIPTION
**Pull request by etsauer**
#### What is this PR About?

This is an attempt to remove the one Travis-CI step we have with one from GitHub actions, as well as moving the uncontained-local-dev image publish step to a process managed in the repo, rather than one that's "hidden" in quay.

#### How should we test or review this PR?

Check that the GitHub Actions all pass. Review code changes.

#### Is there a relevant Trello card or Github issue open for this?

#113 

#### Who would you like to review this?

cc: @redhat-cop/cant-contain-this

'''''

* [x] Have you followed the
[contributing
guidelines?](https://github.com/redhat-cop/uncontained.io/blob/master/CONTRIBUTING.adoc)
* [x] Have you explained what your changes do, and why they add value to
the uncontained.io guides?

'''''

*Please note: we may close your PR without comment if you do not check
the boxes above and provide ALL requested information.*

### CODE REVIEW

File: .github/workflows/container-image-test.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: local-dev-test
+on: [pull_request, push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (skypatrolcpt) [inline .github/workflows/container-image-test.yaml]: Very new to github actions, but seems like `unbuntu-latest` is the default for builds on github actions. Would there be value use a Red Hat based image ?

Comment (sabre1041) [reply .github/workflows/container-image-test.yaml]: These are the runners that GH natively supports:

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idruns-on

You do have the ability to provide your own runner if you would want too

Comment (skypatrolcpt) [reply .github/workflows/container-image-test.yaml]: I assumed something like that. Just wanted to be sure.

**Comment by skypatrolcpt** [comment]
Actions worked on my fork.

Image built locally, using buildah, with no issues.

**Comment by skypatrolcpt** [comment]
Actions worked on my fork.

Image built locally, using buildah, with no issues.

**Pull request merged**